.gitignore

```diff
@@ -0,0 +1,3 @@
+**/
+typing_extensions.py
+zipp.py
```

Comment:
I think it's probably best to delete this file as to not include your personal .gitignore in the edge-functions repo.

Reply:
Can this file just be deleted from the pull request? If not I guess I need to close this PR and do another one.

Reply:
This file can be deleted from the PR.

Just need to delete the file then push up your changes

Reply:
You can add a commit to remove it into your fork, yes. You can also rebase/squash your commits to make a single commit with all your changes if you'd prefer.